go.mod

```diff
@@ -201,3 +201,5 @@ require (
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
+
+replace github.com/mattermost/mattermost-plugin-api => /home/bschumacher/src/mattermost/api
```

Comment:
publish the branch, please and update go.mod?

Reply:
Fixed in a8ce6db215e0bd5b0f454e797f7117dab3903d37